faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -1680,9 +1690,285 @@ class ClientSpec
     result("data").to[Seq[Value]].get should contain theSameElementsInOrderAs expected
   }
 
+  it should "stream return error if it cannot find an instance" in {
+    val err = client.stream(Get(RefV("1234", RefV("spells", Native.Collections)))).failed.futureValue
```

Comment:
According to @erickpintor comments, the query should include just the `Ref`, right?

```scala
val err = client.stream(Ref(Collection("spells"), "1234")).failed.futureValue
```